src/DynamoCoreWpf/Views/SplashScreen/SplashScreen.xaml.cs

```diff
@@ -319,6 +319,7 @@ internal async void SetLabels()
         private void CloseWindow()
         {
            Application.Current.Shutdown();
+           Process.GetCurrentProcess().Kill();
```

Comment:
I think this is too brutal, in process integration like Dynamo Revit will close Revit, most likely we will need changes here

Reply:
Ok, removed. Thanks!

Reply:
@filipeotero If user close Splash Screen from task manager, is it still not closing the current process?